preflight_check/main.py

```diff
@@ -0,0 +1,250 @@
+            task_inst.wait()
+    except task.TaskError as err:
+        task_inst.error(str(err))
+        sys.exit(1)
+
+
+def fence_node(context):
+    """
+    Testcase: fence specific node
+    """
+    if not context.fence_node:
+        return
+
+    task_inst = task.TaskFence(context.fence_node)
+    try:
+        task_inst.pre_check()
+        task_inst.print_header()
+        task_inst.run()
+        task_inst.wait()
+    except task.TaskError as err:
+        task_inst.error(str(err))
+        sys.exit(1)
+
+
+class MyArgParseFormatter(RawTextHelpFormatter):
+    def __init__(self,prog):
+        super(MyArgParseFormatter,self).__init__(prog, max_help_position=50)
+
+
+def parse_argument(context):
+    """
+    Parse argument using argparse
+    """
+    parser = argparse.ArgumentParser(prog=context.process_name,
+                                     description='Cluster preflight check tool set',
+                                     add_help=False,
+                                     formatter_class=MyArgParseFormatter,
+                                     epilog='''
+Log: {}
+Json results: {}
+For each --kill-* testcase, report directory: {}'''.format(context.logfile,
+                                                           context.jsonfile,
+                                                           context.report_path))
+
+    parser.add_argument('-e', '--env-check', dest='env_check', action='store_true',
+                        help='Check environment')
+    parser.add_argument('-c', '--cluster-check', dest='cluster_check', action='store_true',
+                        help='Check cluster state')
+
+    group_mutual = parser.add_mutually_exclusive_group()
+    group_mutual.add_argument('--kill-sbd', dest='sbd', action='store_true',
+                              help='Kill sbd daemon')
+    group_mutual.add_argument('--kill-corosync', dest='corosync', action='store_true',
+                              help='Kill corosync daemon')
+    group_mutual.add_argument('--kill-pacemakerd', dest='pacemakerd', action='store_true',
+                              help='Kill pacemakerd daemon')
+    group_mutual.add_argument('--fence-node', dest='fence_node', metavar='NODE',
+                              help='Fence specific node')
+    group_mutual.add_argument('--split-brain-iptables', dest='sp_iptables', action='store_true',
+                              help='Make split brain by blocking corosync ports')
+    parser.add_argument('-l', '--kill-loop', dest='loop', action='store_true',
+                        help='Kill process in loop')
+
+    other_options = parser.add_argument_group('other options')
+    other_options.add_argument('-y', '--yes', dest='yes', action='store_true',
+                               help='Answer "yes" if asked to run the test')
+    other_options.add_argument('-h', '--help', dest='help', action='store_true',
+                               help='show this help message and exit')
```

Comment:
Start with upper case like others, 'Show this ...`

Reply:
Changed